### ISSUE TITLE
is virtual-repeat ready to be used?

### ISSUE THREAD

**Issue by v3ss0n** [miguelcobain/ember-paper#631]
I am trying to get virtual repeat to work. No errors but i can't get anything to actually display.
Is virtual repeat usable without demo classes on  demo site??
```
{{#paper-list}}
  {{#paper-virtual-repeat model
   as |visibleItems|}}
    {{#each visibleItems as |item|}}

          <div class="md-list-item-text flex">
            <h3>{{item.user}}</h3>
            <h4>{{item.message}}</h4>
          </div>

    {{/each}}
  {{/paper-virtual-repeat}}

{{/paper-list}}
```
![screenshot_20170125_231155](https://cloud.githubusercontent.com/assets/419606/22300253/47834a4c-e355-11e6-96b1-d07b4912a65f.png)

**Comment by v3ss0n** [comment]
inside HTML there are elements there , but it is just not showing. Tried all the demos , reduced down to bare metal and still dosen't work. Give the same set of CSS as Demo and no. 
I am on ember 1.0.14.

**Comment by v3ss0n** [comment]
Found the problem , it dosen't work with percent based layouts
work around is setting virtual-repeat to a class with defined height or width depending on direction :
for percent based , this works
```
.container-main {
  height: 100vh;
  max-height: 100%;
}
```

**Comment by v3ss0n** [comment]
another problem 
Horizontal repate won't work unless virtual-repate classes are altered.

**Comment by jessicabetonico** [comment]
Hi, i have the same problem. Has anyone managed to solve it?

**Comment by miguelcobain** [comment]
I'm inclined to remove it. I feel like users can easily being their own virtual repeat component. There's nothing "material" in it.

**Comment by Subtletree** [comment]
Yeah, using something dedicated to the task sounds better than building something specifically for ember paper

Check out https://github.com/runspired/smoke-and-mirrors

**Comment by v3ss0n** [comment]
@miguelcobain  i agree with your decison , there are many other virtual repate libraries out there .

**Comment by panthony** [comment]
I need something similar to this, shall I add smoke-and-mirrors (or other) instead of investing time in the virtual-repeat component ?

Is there a decision taken on that point ?

**Comment by miguelcobain** [comment]
@panthony please don't use virtual-repeat. I'm inclined to drop it from ember-paper at some point, at least as a public component, certainly.

**Comment by v3ss0n** [comment]
@panthony  please let me know how smoke and mirror works in e-paper . I haven't test it.

**Comment by miguelcobain** [comment]
I never tested it either, but I don't see why it wouldn't work like in any other ember app.

**Comment by v3ss0n** [comment]
I worry it will have problem with percent based/flex layouts.
